source/chapters/user_interface.rst

```diff
@@ -79,13 +87,26 @@ Track Information Display
             to the speed you are tapping, go to :ref:`control-keyboard` for more
             information.
 
-**Time Elapsed/Remaining/Both**
-  By default it displays the total elapsed time in the track up to the
-  millisecond. Clicking on the display switches to the *Time Remaining* view,
-  which indicates how much time is left before the track reaches the end.
-  Clicking the display again, both the elapsed and remaining time are shown
-  simultaneously in the format *elapsed time / remaining time*.
-  You can change the default in :menuselection:`Preferences --> Decks --> Deck options`.
+**Key**
+  .. figure:: ../_static/2.2/key-options.png
+    :align: center
+    :width: 42%
+    :figwidth: 90%
+    :alt: The options displayed when you click on the Key
+    :figclass: pretty-figures
+
+    The Key options.
+
+The key displays the current musical key of the loaded track after pitch shifting. The key is displayed below the BPM (Tempo). This is the same as the key that is shown under the :guilabel:`Key` column heading in the Mixxx library. If it is hidden, you can drag the horizontal scroll bar at the bottom of the Mixxx library section, to the right, and the column will show.
+Clicking the key will expand it to reveal more options:
+
+  * **Pitch Control**: This changes the track pitch independent of the tempo. Left-clicking the :guilabel:`-` button decreases the pitch by one semitone. Right-clicking it decreases the pitch by 10 cents. Left-clicking the :guilabel:`+` button increases the pitch by one semitone. Right-clicking it increases the pitch by 10 cents.
+  * **Sync and Reset Key**: Left-clicking the :guilabel:`MATCH` button sets the key to a pitch that allows harmonic transition from the other track. It requires a detected key on both involved decks. Right-clicking it resets the key to the original track key.
+
+**Cover Art**
+  The cover art of the track loaded into the deck is displayed on the left side of the track. This is the same as the cover art that is shown under the :guilabel:`Cover Art` column heading in the Mixxx library.
```

Comment:
```suggestion
  The cover art of the track loaded into the deck is displayed on the left side of the track. This is the same as the cover art that is shown under the :guilabel:`Cover Art` column heading in the :ref:`Mixxx library <library-tracks>`.
```

Reply:
done.

Reply:
Did you notice you can just click on the 'Commit suggestion' button and let the CI do the chcecking for you and double-check in the preview?
That's why I made those suggestions. No need to apply everything by hand locally...

Reply:
> > > Looking good, just some polishing left.
> > > Please also take care of @Holzhaus 's last review, then we can finally merge IMO.
> > 
> > 
> > Here is what i have in my code for those lines. My width is already at 25% but `figwidth` is at 60%. The new image that i placed here is much smaller.
> > Just to clarify, @Holzhaus do you need me to change the 25% to px?
> > ![image](https://user-images.githubusercontent.com/28389950/97743532-de5f5a00-1af6-11eb-839b-72902ba9dd19.png)
> 
> The question is: Why use a percentage if we use pixel widths for all other raster images? Please either use the pixel width of the image, or check what happens if you remove the `:width:` parameter. I suspect it's unnecessary to specify it at all, because I guess the image will just be displayed fullsize if missing.

Okay. I will try both and commit the one that works best.

Reply:
It turns out the width had no effect on how the image displayed. I have removed it.

Reply:
> Did you notice you can just click on the 'Commit suggestion' button and let the CI do the chcecking for you and double-check in the preview?
> That's why I made those suggestions. No need to apply everything by hand locally...

Oh. Wow. That would definitely make things much easier.  I don't know why i thought i didn't have permission to do this. I will use this button a lot more. Thanks!

Reply:
Yeah, nice feature.
Yo can either commit every single suggestions or create a batch of them.